library/kong

```diff
@@ -13,17 +13,17 @@ GitFetch: refs/tags/3.1.0
 Directory: ubuntu
 Architectures: amd64, arm64v8
 
-Tags: 3.0.1-alpine, 3.0-alpine, 3.0.1, 3.0
-GitCommit: b2057cd3432726c2122873a3740e31a68a03cecd
-GitFetch: refs/tags/3.0.1
+Tags: 3.0.2-alpine, 3.0-alpine, 3.0.2, 3.0
+GitCommit: 2266a0d3e0238e71ed2992ca9ad2453c59af37f6
+GitFetch: refs/tags/3.0.2
 File: Dockerfile.apk
-Architectures: amd64, arm64v8
+Architectures: amd64
```

Comment:
```suggestion
Architectures: amd64, arm64v8
```

Reply:
We do have arm64 for alpine for 3.0.2:
![Screen Shot 2022-12-10 at 11 49 00](https://user-images.githubusercontent.com/310287/206872844-25480374-8059-473f-bf33-b1b6497e62ff.png)
https://hub.docker.com/r/kong/kong/tags?page=1&name=3.0.2